crates/subspace-networking/src/tests/node.rs

```diff
@@ -1,342 +0,0 @@
-use crate::pieces_by_range_handler::{PiecesByRangeRequest, PiecesByRangeResponse};
```

Comment:
Yeah, I was confused why this file was added and this is a good example of the case where amending previous commit would have been less confusing and cleaner instead of adding in one and removing in another.

Reply:
Not always possible without a forced push :(